doc/Navigation/HowTo-ShowFlyout.md

```diff
@@ -6,22 +6,47 @@ This topic walks through using Navigation to display a modal flyout
 
 ## Step-by-steps
 
-
-- Add new page, SamplePage.xaml
-- Add a button "Go to Sample Page" and in the event handler  
-
-**C#**  
-```csharp
+### 1. Displaying flyout from code
+- Add new `Page`, `SamplePage.xaml`, which will be used to display content inside the flyout.
+    ```xml
+    <Page
```

Comment:
I think you'll need to add a line between the text and the callout.
```suggestion
- Add new `Page`, `SamplePage.xaml`, which will be used to display content inside the flyout.

    ```xml
    <Page
```